pkg/validate/validate_test.go

```diff
@@ -324,3 +323,114 @@ func TestValidate(t *testing.T) {
 		}
 	}
 }
+
+func TestValidateUpdateContainerService(t *testing.T) {
+
```

Comment:
no additional newline after `{` or before `}` please

Reply:
it would be nice to have a linter that checked for this stuff